src/sass/misc/_relnote-actions.scss

```diff
@@ -25,7 +25,7 @@
     }
 
     .entry {
-        flex-basis: 400px;
+        flex-basis: 300px;
         padding: 1rem;
```

Comment:
Overall looks good on different resolutions. The only thing I will improve is the content inside the boxes which currently looks very dense.

Before:

![rel-2](https://user-images.githubusercontent.com/2920003/116506544-2101c380-a8db-11eb-9e7a-f21f807ba57c.png)


After applying following CSS it will look little better?
```css
.relnote-actions .entry {
    padding: 1rem 2rem;
}
```

![rel-1](https://user-images.githubusercontent.com/2920003/116506571-2f4fdf80-a8db-11eb-92df-132960c4b11c.png)

Reply:
ack, forgot a responsive entry. committing now.